content/_releases/22/18.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Product Update - v22.18
+description: Details of the product update v22.18 from 5th of May 2022.
+layout: rel
+version: "22.18"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-05-05
+redirect_from:
+  - /releases/22.18.html
+---
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements done in this round.
+
+### Configure login attempts
+
+Now you can configure the number of failed login attempts your users can have before
+locking them out of the system. We added a new variable `MAX_LOGIN_ATTEMPTS` to the
+HELM3 secrets chart. Set this variable any reasonable number for your system, otherwise
+it defaults to 5. Here is how we calculate this number:
+
+*   In case you don't have 2FA enabled, this is the number of failed login attempts.
```

Comment:
2FA -> 2-factor Authentication (2FA) N.B. (full version at first use followed by abbreviation in parentheses - thereafter just the abbr.

In case you don't have 2-factor Authentication (2FA) enabled, this is the number of failed login attempts due to incorrect Username and/or password.

Reply:
> due to incorrect Username
@KMobbs 
no. Username in our case is an email. so, if user put the wrong email we will answer that given user doesn't exist. This couner increased only if email exists in our platform but password is wrong. So we identify user by email and increase the value of failed logins for this user